src/StatusQ/Components/StatusListItem.qml

```diff
@@ -144,7 +169,7 @@ Rectangle {
                 height: visible ? contentHeight : 0
                 wrapMode: Text.WrapAtWordBoundaryOrAnywhere
                 anchors.left: parent.left
-                anchors.right: !statusListItem.titleAsideText ? parent.right : undefined
+                anchors.right: !statusListItem.titleAsideText && !titleIconsRow.visible ? parent.right : undefined
```

Comment:
Here there is an issue that I propose to address in a separate task as it afects `StatusListItem` component and I've seen the same issue in `StatusMessageHeader.qml` component as when the `anchors.right` property is set as `undefined` neither `titleIconsRow` nor `titleAsideText` are correctly positioned and the `statusListItemTitle` comes out of the expected boundaries.

Reply:
Here the created issue [#540](https://app.zenhub.com/workspaces/status-desktop-5ea70d29c12ee928df5ae848/issues/status-im/statusq/540)